packages/astro-imagetools/integration/index.js

```diff
@@ -1,15 +1,12 @@
 // @ts-check
 import fs from "node:fs";
-import path from "node:path";
+import { posix as path, resolve } from "path";
```

Comment:
As the other imports are prefixed with `node:`, can you prefix it too to avoid any kind of compatibility issues?